dist/stat_data.py

```diff
@@ -526,6 +526,9 @@ def __init__(self, name, desc, flags=''):
     TxnStat('txn_prepare_active', 'prepared transactions currently active'),
     TxnStat('txn_prepare_commit', 'prepared transactions committed'),
     TxnStat('txn_prepare_rollback', 'prepared transactions rolled back'),
+    TxnStat('txn_prepared_updates_active', 'prepared active updates'),
```

Comment:
"active updates" suggest that this many updates are prepared at the moment, whereas this counter measures how many updates we have marked as prepared until now (a commit / rollback doesnt decrease this count so these are not active at the moment). Can we call it something on the lines of :
```TxnStat('txn_prepared_updates', 'prepared updates count')